app/services/rollover_service.rb

```diff
@@ -1,9 +1,12 @@
 class RolloverService
   include ServicePattern
 
-  def initialize(provider_codes:, force: false)
+  def initialize(provider_codes:, force: nil, source_recruitment_cycle: RecruitmentCycle.current_recruitment_cycle, target_recruitment_cycle: RecruitmentCycle.next_recruitment_cycle, logger: Rails.logger)
```

Comment:
```suggestion
  def initialize(provider_codes:, force:, source_recruitment_cycle: RecruitmentCycle.current_recruitment_cycle, target_recruitment_cycle: RecruitmentCycle.next_recruitment_cycle, logger: Rails.logger)
``` 
Also I don't think we need to inject the logger here do we? We'll always be using `Rails.logger`

Reply:
It was getting really confusing with 

All 3 of these was getting used
`puts`
`prints`
`Logger.new(STDOUT)`

as well as all the noise

![image](https://user-images.githubusercontent.com/470137/142609072-6532e19c-45cf-4e2f-a743-0d662b55e2f5.png)

![image](https://user-images.githubusercontent.com/470137/142609544-4ecad356-2e28-436d-934f-7da35e7deebc.png)

And since reading the log will be an actual thing to be done, it should be more extensive tested, hence injection.